src/docusaurus.config.js

```diff
@@ -20,14 +20,16 @@ const config = {
   },
   plugins: [
     'docusaurus-plugin-image-zoom',
-    [
-      '@docusaurus/plugin-google-gtag',
-      /** @type {import('@docusaurus/plugin-google-gtag').Options} */
-      ({
-        trackingID: 'G-XN33JD9H3F',
-        anonymizeIP: true,
-      }),
-    ],
+    ...(process.env.NODE_ENV === 'production' ? [
```

Comment:
Is this actually fixing a bug or something? I don't see an error when running locally.